Fix : Update the documentation for HAL and FAL consistent with labVIEW (#6)
NI-Measurement-Plug-Ins/abstraction-layer-python: docs/HAL in Measurement Plug-In.md

# Creating a HAL for Use in Measurement Plug-ins

- [Creating a HAL for Use in Measurement Plug-ins](#creating-a-hal-for-use-in-measurement-plug-ins)
  - [What is HAL](#what-is-hal)
  - [Pre-requisites](#pre-requisites)
  - [Steps to implement HAL for a new instrument type](#steps-to-implement-hal-for-a-new-instrument-type)
  - [Directory structure of HAL](#directory-structure-of-hal)
  - [Migrate the existing instrument class to Measurement Plug-In](#migrate-the-existing-instrument-class-to-measurement-plug-in)
  - [Note](#note)

> ## What is HAL

Reviewer: ```suggestion
## What is HAL?
```

```

## Migrate the existing instrument class to Measurement Plug-In

- Create a measurement plug-in by following the steps mentioned in
  [Developing a measurement plug-in with python](https://github.com/ni/measurement-plugin-python?tab=readme-ov-file#developing-measurements-quick-start).

> - Copy the existing instrument classes to the project and follow the steps from
>   [Steps to create a new HAL based measurement](#steps-to-implement-hal-for-a-new-instrument-type) section
>   for migrating the existing instrument classes to measurement plug-in.

Reviewer: ```suggestion
- Copy the existing HAL classes and modules by following the steps from [Steps to create a new HAL based measurement](#steps-to-implement-hal-for-a-new-instrument-type) to migrate the existing HAL implementation.
```
